src/pages/journal/journal-components/journal-slot/journal-slot.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'journal-slot',
+  templateUrl: 'journal-slot.html',
+  inputs: ['slot']
+})
+export class JournalSlotComponent {
+  @Input()
+  slot: any;
```

Comment:
Can we get a type on this? Something from the JSON Schema?

Reply:
At the moment the automated JSON schema doesn't generate an interface for testSlot so not able to use a generated type as yet.

Reply:
I think there should be a ticket in jira which will require us to implement the data structure. What are your thoughts?

Reply:
I believe there are plans to move the JSON schema and auto generation to a separate repository, that can be brought in where needed.  I think at that point it should either have the auto generation fixed so that it generates more interfaces, or be done by hand.